CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# How to contribute
+
+shipyard is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers and other resources to make getting your contribution into shipyard easier.
```

Comment:
not `shipyard`, right?

Reply:
sure! fixed (there were a couple of them over there..)